extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -194,8 +200,29 @@ export class Placement {
    */
   placeAd(baseAttributes, sizing, adTracker, isResponsiveEnabled) {
     return this.getEstimatedPosition().then((yPosition) => {
-      // TODO(powerivq@) Remove this after fixing ad resizing
-      if (this.ampdoc.win./*OK*/ scrollY > yPosition) {
+      // TODO(powerivq@) Remove this after finishing the experiment
+      const controlBranch = '31060868';
+      const expBranch = '31060869';
+      const holdbackExp = isExperimentOn(
+        this.ampdoc.win,
+        'auto-ads-no-insertion-above'
+      );
+      if (holdbackExp) {
+        const expInfoList = /** @type {!Array<!../../../experiments.ExperimentInfo>} */ ([
+          {
+            experimentId: 'auto-ads-no-insertion-above',
+            isTrafficEligible: () => true,
+            branches: [controlBranch, expBranch],
+          },
+        ]);
+        randomlySelectUnsetExperiments(this.ampdoc.win, expInfoList);
+      }
+      if (
+        (!holdbackExp ||
+          getExperimentBranch(this.ampdoc.win, 'auto-ads-no-insertion-above') ==
+            expBranch) &&
```

Comment:
Should this be control?

Reply:
Experiment branch points to the current behavior and control points to the old behavior.

Reply:
👍